data/pokemon/palettes.asm

```diff
@@ -5,6 +5,8 @@ PokemonPalettes:
 ; only the middle two colors are included, not black or white.
 ; Shiny palettes are defined directly, not generated.
 
+	table_width PAL_COLOR_SIZE * 2 * 2, PokemonPalettes
```

Comment:
Again, `* 2 * 2` might be confusing and worth a comment (although this one's clearer).